components/tyk-dev-portal/values.yaml

```
@@ -17,6 +17,10 @@ global:
     email: default@example.com
     # password of Developer Portal admin
     password: 123456
+    # firstname of Developer Portal admin
+    firstName: "admin"
+    # lastName of Developer Portal admin
+    lastName: "tyk"
```

1 suggestion, can we make it `user` to make the field consistent with other charts?
```suggestion
    lastName: "user"
```